icommand.lisp

```diff
@@ -38,22 +38,22 @@
   (message "~s finished" name)
   (pop-top-map))
 
-(defmacro deficommand (name key-bindings &key pre-start-fn cleanup-fn)
+(defmacro deficommand (name (&key bootstrap-fn cleanup-fn) &body key-bindings)
   "Declare an interactive command." ;; TODO
   (let* ((command (if (listp name) (car name) name))
-         (exit-command (concatenate 'string "exit-" (symbol-name command)))
-         (m-name (gensym "m")))
-    `(let ((,m-name (make-sparse-keymap)))
+         (exit-command (concatenate 'string "EXIT-" (symbol-name command)))
```

Comment:
Here we can use format instead`(format nil "EXIT-~A" command)`, which doesn't need thecall to symbol-name. (Or alexandria:symbolicate which also saves as the call to intern below)